### ISSUE TITLE
Attempt to index field 'args' (a nil value) upon setting custom bar to track Value for Spell Tooltip (of a totem)

### ISSUE THREAD

**Issue by Lannakin** [Dicebar/Raven#44]
I was setting up a bar to track the progress of Cloudburst Totem while it was up, so I'd know when it was going to expire.

```
2x Raven/Bars.lua:1053: attempt to index field 'args' (a nil value)
[string "@Raven/Bars.lua"]:1053: in function `GetTooltipNumber'
[string "@Raven/Values.lua"]:660: in function `f'
[string "@Raven/Bars.lua"]:2001: in function <Raven/Bars.lua:1898>
[string "@Raven/Bars.lua"]:2146: in function `UpdateBars'
[string "@Raven/Main.lua"]:1266: in function `Update'
[string "@Raven/Main.lua"]:924: in function <Raven/Main.lua:920>

Locals:
self = <table> {
 CheckTalent = <function> defined @Raven/Conditions.lua:398
 testOrder = <table> {
 }
 HexColor = <function> defined @Raven/Profile.lua:43
 UpdateInCombatBar = <function> defined @Raven/Cancel.lua:66
 modules = <table> {
 }
 Nest_SetShine = <function> defined @Raven/Nest.lua:1303
 GetValueFormat = <function> defined @Raven/Values.lua:833
 ReleaseBarGroup = <function> defined @Raven/Bars.lua:723
 CopyTable = <function> defined @Raven/Profile.lua:61
 ResetColorDefault = <function> defined @Raven/Profile.lua:452
 Nest_SetBarGroupLock = <function> defined @Raven/Nest.lua:973
 Nest_FormatTime = <function> defined @Raven/Nest.lua:1873
 SetInCombatBar = <function> defined @Raven/Cancel.lua:203
 LocalSpellNames = <table> {
 }
 GetExpireColor = <function> defined @Raven/Profile.lua:669
 UpdateInternalCooldowns = <function> defined @Raven/Main.lua:2119
 SetExpireColor = <function> defined @Raven/Profile.lua:657
 DefaultProfile = <table> {
 }
 ResetColorDefaults = <function> defined @Raven/Profile.lua:471
 RemoveMatchingTrackers = <function> defined @Raven/Main.lua:401
 Nest_CreateBar = <function> defined @Raven/Nest.lua:1074
 CheckSpec = <function> defined @Raven/Conditions.lua:453
 SetEnabledState = <function> defined @BartrubySummonPet/libs/AceAddon-3.0/AceAddon-3.0.lua:420
 FinalizeBars = <function> defined @Raven/Bars.lua:161
 mountSpells = <table> {
 }
 Nest_SetLabelColor = <function> defined @Raven/Nest.lua:1279
 LoadCustomBars = <function> defined @Raven/Bars.lua:528
 Nest_DeleteAllBars = <function> defined @Raven/Nest.lua:1243
 Nest_AdjustColor = <function> defined @Raven/Nest.lua:1344
 UpdateTrackers = <function> defined @Raven/Main.lua:1870
 ldbi = <table> {
 }
 Nest_RegisterCallbacks = <function> defined @Raven/Nest.lua:683
 UnregisterAllEvents = <function> defined @BartrubySummonPet/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
 InitializeBarGroup = <function> defined @Raven/Bars.lua:494
 Nest_SetBarGroupBorder = <function> defined @Raven/Nest.lua:898
 Nest_GetBarGroups = <function> defined @Raven/Nest.lua:796
 HideBars = <function> defined @Raven/Bars.lua:175
 OptionsPanel = <function> defined @Raven/Main.lua:1105
 Nest_SetBarGroupBackdrop = <function> defined @Raven/Nest.lua:906
 DetectInternalCooldown = <function> defined @Raven/Main.lua:2101
 Nest_SetColors = <function> defined @Raven/Nest.lua:1262
 warlockCreatures = <table> {
 }
 InitializeSpellIDs = <function> defined @Raven/Profile.lua:141
 GetName = <function> defined @BartrubySummonPet/libs/AceAddon-3.0/AceAddon-3.0.lua:275
 InitializeBuffTooltip = <function> defined @Raven/Main.lua:1445
 name = "Raven"
 ColorPalette = <table> {
 }
 SetConditionDefaults = <function> defined @Raven/Conditions.lua:937
 Nest_SetAlpha = <function> defined @Raven/Nest.lua:1294
 SaveBarGroupSettings = <function> defined @Raven/Bars.lua:525
 Print = <function> defined @BartrubySummonPet/libs/AceConsole-3.0/AceConsole-3.0.lua:50
 Nest_SetTimeColor = <function> defined @Raven/Nest.lua:1284
 Nest_DeleteBar = <function> defined @Raven/Nest.lua:1179
 ExpansionIsOrBelow = <function> defined @Raven/Main.lua:43
 UpdateSpellEffects = <function> defined @Raven/Main.lua:1718
 Debug = <function> defined @Raven/Main.lua:206
 Nest_IsTimer = <function> defined @Raven/Nest.
```

**Comment by Dicebar** [comment]
Heya, thanks for the report.

I couldn't reproduce the bug locally, but it looks very similar to a bug we fixed elsewhere for 10.1. It should be fixed in the next release.

**Comment by Lannakin** [comment]
Thank you, I look forward to this.

**Comment by Dicebar** [comment]
That release was two days ago. If you're still looking forward to the fix...did it not work? Are you using the latest version?

**Comment by Dicebar** [comment]
I'm closing this issue due to inactivity. If the problem has not been solved, feel free to reopen it.

**Comment by Lannakin** [comment]
I'm sorry for the slow response.  I had ended up setting up a weakaura to do this instead and when I went to respond before, I couldn't remember whether I'd actually retested or not (and when I went to check, I kept forgetting what I was doing somehow).

I went through a very convoluted attempt to display the totem timer because I thought I needed to make a Value type custom bar that monitors a Spell Tooltip because I forgot how I made it last time.  I did this for  *at least* an hour, and then realized maybe it should be a Buff type bar.

**It works fine now.  Thanks!**
